localstack/http/request.py

```diff
@@ -78,7 +81,17 @@ def form(self):
 
     @property
     def values(self):
-        raise NotImplementedError
+        """Naive implementation to parse and return form data.
+
+        The property is invoked by Moto ELB to get the ELB API version in
+        `moto.elb.urls.api_version_elb_backend()`
+        """
+        params = to_str(self.data).split("&")
+        output = {}
+        for param in params:
+            key, value = param.split("=")
+            output[key] = value
+        return output
```

Comment:
this is pretty much what `self.args` from `_SansIORequest` does. i think in the case of `values` we can just copy the code from werkzeug, since `values` itself does no IO, it just combines `args` with `form` data. https://github.com/pallets/werkzeug/blob/c7ae2fea4fb229ffd71187c2b665874c91b96277/src/werkzeug/wrappers/request.py#L432-L440

however, since we don't have `form` implemented yet, i may just comment out the line and add a TODO https://github.com/pallets/werkzeug/blob/c7ae2fea4fb229ffd71187c2b665874c91b96277/src/werkzeug/wrappers/request.py#L448